cluster/examples/kubernetes/ceph/object-bucket-claim-retain.yaml

```diff
@@ -11,3 +11,7 @@ spec:
   #bucketName: 
   generateBucketName: ceph-bkt
   storageClassName: rook-ceph-retain-bucket
+  additionalConfig:
```

Comment:
@thotz Let's test these new quota settings in the integration tests by always setting the quotas in the OBC [here](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go#L2265-L2277).